site/docs/v1/tech/apis/authentication.adoc

```diff
@@ -114,6 +152,9 @@ image::create-api-key.png[Create an API Key,width=1200,role=shadowed]
 ==== Form Fields
```

Comment:
For `create-api-key.png`, please re-take this screenshot with the scroll bar turned off. Instructions are here: https://github.com/FusionAuth/fusionauth-site/blob/master/DocsDevREADME.md#sizing-window-for-screenshots 

Please remove `shadowed` from the roles.

Please ensure that you have resized your browser to the correct size (it looks like you haven't as the size is 1600x1450, compare with https://fusionauth.io/assets/img/docs/installation-guides/cloud/provisioning-select-size-url.png which is 1600x1627).

The image guidelines are documented here: https://github.com/FusionAuth/fusionauth-site/blob/master/DocsDevREADME.md#sizing-window-for-screenshots It took me a few times to get them right too :)